control-plane/agents/Cargo.toml

```diff
@@ -46,6 +50,7 @@ indexmap = "1.8.0"
 k8s-openapi = { version = "0.13.0", default-features = false, features = ["v1_20"] }
 kube = { version = "0.60.0", features = ["derive" ] }
 anyhow = "1.0.44"
+clap =  { version = "3.1.5", features = ["color", "derive"] }
```

Comment:
nit: I'd prefer to either use structopt since it's what the other agents use or instead change all agents to clap (latter would be my preference :))